tutorials/polkadot-sdk/build-a-blockchain/build-a-local-blockchain.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Build a Local Solochain
+description: Learn to compile and launch a local blockchain node using Polkadot SDK. Build, run, and interact with a pre-configured node template.
+---
+
+# Build a Local Solochain
+
+## Introduction
+
+[Polkadot SDK](https://github.com/paritytech/polkadot-sdk){target=\_blank} offers a versatile and extensible blockchain development framework, enabling you to create custom blockchains tailored to your specific application or business requirements. 
+
+This tutorial guides you through compiling and launching a standalone blockchain node using the [Polkadot SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank}. You'll create a fully functional chain that operates independently, without connections to a relay chain or parachain.
+
+The node template provides a pre-configured, functional single-node blockchain you can run in your local development environment. It includes several key components, such as user accounts and account balances.
+
+These predefined elements allow you to experiment with common blockchain operations without requiring initial template modifications.
+In this tutorial, you will:
+
+- Build and start a local blockchain node using the node template
+- Explore how to use a front-end interface to:
+    - View information about blockchain activity
+    - Submit a transaction
+
+By the end of this tutorial, you'll have a working local solochain and understand how to interact with it, setting the foundation for further customization and development.
+
+## Prerequisites
+
+To get started with the node template, you'll need to have the following set up on your development machine first:
+
+- Rust Installation - the node template is written in [Rust](https://www.rust-lang.org/){target=\_blank}, so you'll need to have it installed and configured on your system. Refer to the [Installation]() section for step-by-step instructions on setting up your development environment
```

Comment:
```suggestion
- Rust Installation - the node template is written in [Rust](https://www.rust-lang.org/){target=\_blank}, so you'll need to have it installed and configured on your system. Refer to the [Installation]() section for step-by-step instructions on setting up your development environment
```
```suggestion
- Rust installation - the node template is written in [Rust](https://www.rust-lang.org/){target=\_blank}, so you'll need to have it installed and configured on your system. Refer to the [Installation]() section for step-by-step instructions on setting up your development environment
```

Oops, I made a comment and then resolved it haha so here is another comment, because this still needs to be done 😬